r/tools/nixlibs.R

```diff
@@ -77,7 +80,7 @@ download_binary <- function(os = identify_os()) {
         cat("**** If installation fails, retry after installing those system requirements\n")
       }
     } else {
-      cat(sprintf("*** No C++ binaries found for %s\n", os))
+      cat(sprintf("*** No Arrow C++ binary found for version %s on %s\n", VERSION, os))
```

Comment:
```suggestion
      cat(sprintf("*** No libarrow binary found for version %s on %s\n", VERSION, os))
```
Small change - feel free to reject - but I've been referring to Arrow C++ as `libarrow` in a lot of the docs + it also then matches up with the env var.